package.json

```diff
@@ -9,11 +9,12 @@
     "node": "^4 || ^5"
   },
   "devDependencies": {
+    "SMS": "file:./apps/sms",
     "adm-zip": "^0.4.7",
     "app-root-path": "^1.0.0",
     "archiver": "^0.16.0",
     "async": "^1.5.0",
-    "babel-eslint": "^4.1.5",
+    "babel-eslint": "^5.0.0-beta6",
```

Comment:
I think you only needed to run "npm install" from the root directory. Can you please try this again instead of upgrading babel-eslint ?
Because you did the change I think it's better (to be sure it's working) if you run `rm -rf node_modules && make node_modules`, to be sure your node_modules directory is in a right state.